### ISSUE TITLE
1.10 Agent welcome email selection is missing

### ISSUE THREAD

**Issue by troycarpenter** [osTicket/osTicket-1.8#2870]
In 1.9.12, when adding a new agent there is a checkbox that says for Welcome Email when creating the agent account.  That selection seems to be missing in 1.10.  If it moved to another screen, I can't find it.

**Comment by troycarpenter** [comment]
To make this work, I added the following in include/staff/staff.inc.php at line 175:
```
<?php if ($action=='create') { ?>
    <label class="checkbox">
        <input type="checkbox" name="welcome_email" id="welcome-email" checked="checked"/>
        <?php echo __('Send Welcome Email'); ?>
    </label>
<?php } ?>
```
It appears the backend code still understands how to send the email.  The first line assures the checkbox only shows up when adding an agent.  Also, this assumes you want to send the welcome emails by default as it automatically checks the box.

**Comment by xdreamarun** [comment]
we have added same code at same place.
we are using osticket version 1.10
Fields is displaying while create the agent.
But no email is going to agent.

**Comment by luismariveros** [comment]
I just installed osTicket Version v1.10 (901e5ea) and have the same problem... With the code of @troycarpenter it worked... I added at line 195

**Comment by lzantal** [comment]
Thanks for the code @troycarpenter. This fixed my installation as well. New user to osTicket so I didn't know where to start to enable it. Whats weird is that it sent it out twice before but then it suddenly stopped. Thanks again.

**Comment by akshaynikhare** [comment]
@troycarpenter  the code should be inserted at line 198 with generate better output  
![image](https://user-images.githubusercontent.com/9725336/70790040-3c3d6300-1dba-11ea-9bb3-9d6bc9a199c8.png)

as this  
![image](https://user-images.githubusercontent.com/9725336/70790071-4a8b7f00-1dba-11ea-82c2-78609fd6fde3.png)

**Comment by ManZosh** [comment]
Thanks @akshaynikhare this helped big time! I'm now only wondering why this issue was opened just about 4 years ago and hasn't yet been committed to the code base after 4 versions?

**Comment by JediKev** [comment]
@ManZosh 

Because this is the incorrect way of addressing the issue. The correct fix is found below and should be added to the next release once finalized:
- https://github.com/osTicket/osTicket/pull/5308

Adding the checkbox back is incorrect as we now determine the Welcome Email on wether or not you set a password for the new Agent and if the Authentication Backend is external (ie. Active Directory). If a password is not set and the Authentication Backend is Local then we send a Welcome Email so the Agent can set their password and gain access to the system. If no password is set and the Authentication Backend is AD for example, we do not send the welcome email as they do not need to set their password as they use their AD credentials to login. The Help Tip for the old checkbox used to say `If unchecked, you will need to set password and communicate the log-in information to the new staff.` so if a password is set it's understood for the creator of the account to communicate the login information to the Agent in question.

Cheers.

**Comment by akshaynikhare** [comment]
@JediKev so you meen ,
This is some ware relate to that active directory plugin of ostickets

**Comment by JediKev** [comment]
@akshaynikhare 

No, I just mean that the previous fix suggested above is an incorrect approach to addressing the issue.

Cheers.
